frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -38,45 +38,44 @@ function isValidClusterNameFormat(name: string) {
 }
 
 export default function SettingsCluster() {
-  const cluster = useCluster();
```

Comment:
If we keep the cluster route for the settings, c/mycluster/settings, we will be able to determine the cluster we should display for the settings directly from the context, and the logic of the component would be much closer to what we had before as well.

Reply:
Added back in